src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,100 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import cn from 'classnames';
+import { TodosContext } from '../TodoContext';
+import { addComment } from '../utils/api';
+import { Comment } from '../types/Comment';
+
+interface Props {
+  comments: Comment[];
+  setComments: (q: Comment[]) => void;
+}
+
+export const NewCommentForm: React.FC<Props> = ({ comments, setComments }) => {
+  const [commentName, setCommentName] = useState('');
+  const [commentEmail, setCommentEmail] = useState('');
+  const [commentText, setCommentText] = useState('');
```

Comment:
I recommend you to make it as one instance 
```suggestion
  const [comment, setComment] = useState({
    name: '',
    email: '',
    text: '',
  });
```